### PR TITLE
Fix to languages

### PR DESCRIPTION
**Pull request by germanngc**
Removed all the Cookie language set and moved into SetLocale middleware created by @MaxMancera1 

This will require to clear the views and routes:

```bash
docker exec ninacodemx-app php artisan route:clear
docker exec ninacodemx-app php artisan view:clear
```

### CODE REVIEW

File: lang/es/pages/articles.php [hunk 1]
```diff
@@ -21,8 +21,8 @@
 	],
     'hero' => [
 		'h1' => 'Articulos',
-		'h2' => 'Cuenta con nosotros para llevar tu negocio al siguiente nivel',
-		'p' => 'Nuestros servicios están enfocados en proveer soluciones a la medida a cada negocio, ya sea que necesites una página web, una aplicación móvil, o un sistema de gestión empresarial, nosotros te ayudamos a lograrlo.',
+		'h2' => 'Ecneuntra información relevante para tu negocio.',
```
Comment (MaxMancera1) [inline lang/es/pages/articles.php]:
```suggestion
		'h2' => 'Encuentra información relevante para tu negocio.',
```

**Review by MaxMancera1 — CHANGES_REQUESTED** [review]

**Review by germanngc — COMMENTED** [review]
Self review

**Review by germanngc — COMMENTED** [review]
Self review

**Review by MaxMancera1 — APPROVED** [review]
CHECKED

**Pull request merged**